spec/label_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../book/label'
+require_relative '../book/book'
+
+describe Label do
+  context 'To create a new instance of the Label class we must ensure the following' do
+    it 'Return a Label object including title and color via constructor method' do
+      @title = 'Fiction'
+      @color = 'Blue'
+      @label = Label.new(@title, @color)
+      expect(@label.title).to eql @title
+      expect(@label.color).to eql @color
```

Comment:
I would suggest you combine all the `except` into one `expect ` to follow the AAA principle